ts/src/patch/cosmos/base/v1beta1/coin.ts

```diff
@@ -1,20 +1,18 @@
+import { Decimal } from "@cosmjs/math";
 import * as minimal from "protobufjs/minimal";
 import { Reader } from "protobufjs/minimal";
 
 import * as coin from "../../../../generated/cosmos/base/v1beta1/coin.original";
 import { DecCoin } from "../../../../generated/cosmos/base/v1beta1/coin.original";
 
 const originalEncode = coin.DecCoin.encode;
+const PRECISION = 18;
```

Comment:
Lets add a comment the value has been derrived from here https://github.com/cosmos/cosmos-sdk/blob/25b14c3caa2ecdc99840dbb88fdb3a2d8ac02158/math/dec.go#L21